webextensions/manifest/options_ui.json

```diff
@@ -79,7 +79,9 @@
               },
               "edge": "mirror",
               "firefox": {
-                "version_added": "48"
+                "alternative_name": "options_page",
+                "version_added": "48",
+                "notes": "An alias to <code>options_page</code> from Firefox 126. When set, <code>options_page</code> is ignored."
```

Comment:
@rebloor given that `"alternative_name": "options_page"` would only apply on version >= 126 should we change this to an array?

something like:
```
"firefox": [
  { "version_added": "48" },
  {
    "alternative_name": "options_page",
    "version_added": "126",
    "note": "..."
  } 
]
```

May only concern about turning this into an array is the side-effect it may have in the way the addons-linter uses this compat data internally.

I'd personally be ok to only add the alternative_name and notes on the options_page.json in this PR, just to stay on the safe side and consider this addition eventually as a followup, so that we can avoid to block this on the detail related to the addons-linter.

Unless @Rob--W feels more strongly about including this note.

Reply:
Let's drop the reference to options_page from options_ui. From the compatibility perspective options_page is not relevant when options_ui is set.

Reply:
:+1: Fine by me too, let's keep this simple.